ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py

```diff
@@ -1743,6 +1743,82 @@ def test_jax_lax_dot(
     )
 
 
+@st.composite
+def _get_dtype_inputs_for_batch_matmul(draw):
+    dim_size = draw(helpers.ints(min_value=1, max_value=4))
+    batch_size = draw(helpers.ints(min_value=1, max_value=3))
+    dtype = draw(helpers.get_dtypes("numeric", index=1, full=False))
```

Comment:
Also, please use all "valid" dtypes here instead of "numeric", and add `@with_supported_dtypes` to the function to specify that it only supports numeric inputs.